scripts/schemas_check.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import re
+import requests
+
+from requests.exceptions import RequestException
+
+logger = logging.getLogger(__name__)
+
+try:
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-questionnaire-schemas/releases"
+    )
+    if response.status_code == 200:
+        latest_tag = response.json()[0]["tag_name"]
+        f = open("Makefile", "r")
+        file = f.readlines()
+        for i, line in enumerate(file):
+            if "onsdigital/eq-questionnaire-schemas" in line:
+                ref = re.compile(r"v\d.\d.\d")
+                mo = ref.search(file[i])
+                version = mo.group()
+                break
```

Comment:
This might be clearer as a separate method. If not, the `version` var need to also be defined outside the for loop.

Reply:
We don't need regex either, we could just do `line.split(' ')[-1]` to return the version.

Reply:
All done, thanks👍

Reply:
Done, thanks!